databases/models/Channels.js

```diff
@@ -0,0 +1,46 @@
+"use strict";
+const { Model } = require("sequelize");
+module.exports = (sequelize, DataTypes) => {
+  class Channels extends Model {
+    /**
+     * Helper method for defining associations.
+     * This method is not a part of Sequelize lifecycle.
+     * The `models/index` file will call this method automatically.
+     */
+    static associate(models) {
+      // define association here
+    }
+  }
+  Channels.init(
+    {
+        id: {
+            type: DataTypes.INTEGER,
+            allowNull: false,
+            primaryKey: true,
+            autoIncrement: true
+        },
+        channel_id:{
+            type: DataTypes.UUID,
+            allowNull: false,
+        },
+        created_by: {
+            type: DataTypes.STRING
+        },
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `created_by` field does not have a `allowNull` property. If this field is optional, it's fine. However, if it's required, you should add `allowNull: false`.

```diff
+        created_by: {
+            type: DataTypes.STRING,
+            allowNull: false,
+        },
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->